src/Cake.Svn/SvnAliases.Info.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+using Cake.Svn.Info;
+
+namespace Cake.Svn
+{
+    public static partial class SvnAliases
+    {
+        /// <summary>
+        /// Check if the <paramref name="directoryPath"/> is controlled by Subversion.
+        /// </summary>
+        /// <param name="context">The Cake context.</param>
+        /// <param name="directoryPath">The directory.</param>
+        /// <returns>
+        /// <c>true</c> if the the directory <paramref name="directoryPath"/> is controlled by Subversion. 
+        /// Otherwise <c>false</c> will be returned.
+        /// </returns>
+        /// <example>
+        /// Check if the <paramref name="directoryPath"/> is controlled by Subversion.
+        /// <code>
+        /// <![CDATA[
+        ///     var isWorkingCopy = IsWorkingCopy(@"C:\project\src\newfolder\");
+        ///
+        ///     Verbose("The directory is a working copy: {0}", isWorkingCopy);
+        /// ]]>
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        [CakeAliasCategory("Info")]
+        [CakeNamespaceImport("Cake.Svn.Info")]
+        public static bool IsWorkingCopy(this ICakeContext context, DirectoryPath directoryPath)
```

Comment:
This alias name is ambigous with the FilePath overload. Also it has a potentical risk of naming conflicts with other addins.

I suggest rename to `IsSvnWorkingCopy`